digital-twins/jsonld-contexts/observed_attributes.jsonld

```diff
@@ -0,0 +1,9 @@
+  {
+    "@context": {
+        "name": "https://uri.fiware.org/ns/data-models#name",
+        "description": "https://uri.fiware.org/ns/data-models#description",
```

Comment:
those 2 terms are already defined in the core context (with IRIs from well known ontologies, you can use them)

Reply:
If i'm not mistaken, only description is defined in the core context 1.3 ("name" does not appear here => https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld)

Description is defined with: http://purl.org/dc/terms/description

Also I'm not able to find those "well known ontologies" from IRIs. Do you know a particular website or something? What about https://schema.org/name ?

Reply:
Weird, I was pretty sure that `name` was defined in the core context and mapped to https://schema.org/name but no. And the ontology is schema.org.

Description comes from Dublin Core: https://www.dublincore.org/specifications/dublin-core/dcmi-terms/